jams/__init__.py

```diff
@@ -19,11 +20,17 @@
 
 
 # Populate the namespace mapping
-for ns in chain(*map(lambda p: p.rglob('*.json'), resources.files('jams.schemata.namespaces').iterdir())):
-    schema.add_namespace(ns)
+if sys.version_info < (3, 10):
+    from pkg_resources import resource_filename
```

Comment:
thought that leaving the import statement here might be better than at the beginning of the code to avoid deprecation warning for newer python versions?

Reply:
Yeah that should be fine.